rvv-intrinsic-rfc.md

```diff
@@ -407,9 +409,9 @@ For reduction intrinsics which may only keep the tail elements.
 |--- |--- |--- |---
 |No  | No | N/A| `vredsum_vs_i8m1_ta(vs1, vs2, vl)`
 |No  | Yes| N/A| `vredsum_vs_i8m1_tu(vd, vs1, vs2, vl)` and `vredsum_vs_i8m1(vd, vs1, vs2, vl)`
-|Yes | No | No | `vredsum_vs_i8m1_tama(mask, vs1, vs2, vl)`
+|Yes | No | No | `vredsum_vs_i8m1_m_ta(mask, vs1, vs2, vl)`
 |Yes | No | Yes| `vredsum_vv_i8m1_m(mask, vd, vs1, vs2, vl)` if vd is `vundefined()`
-|Yes | Yes| No | `vredsum_vs_i8m1_tuma(mask, vd, vs1, vs2, vl)`
+|Yes | Yes| No | `vredsum_vs_i8m1_m_tu(mask, vd, vs1, vs2, vl)`
 |Yes | Yes| Yes| `vredsum_vs_i8m1_m(mask, vd, vs1, vs2, vl)`
```

Comment:
Maybe like this?
```
|Masked?  | TU? | MU? | Intrinsic
|--- |--- |--- |---
|No  | No | N/A | `vredsum_vs_i8m1_ta(vs1, vs2, vl)`
|No  | Yes| N/A | `vredsum_vs_i8m1_tu(vd, vs1, vs2, vl)` and `vredsum_vs_i8m1(vd, vs1, vs2, vl)`
|Yes | No | N/A | `vredsum_vs_i8m1_m_ta(mask, vs1, vs2, vl)`
|Yes | Yes| N/A | `vredsum_vs_i8m1_m_tu(mask, vd, vs1, vs2, vl)` and `vredsum_vs_i8m1_m(mask, vd, vs1, vs2, vl)`
```

Reply:
I realized why you write that because we have this in other place
```
|Yes | No | Yes| `vadd_vv_i8m1_tamu(mask, merge, vs2, vs1, vl)` and `vadd_vv_i8m1_m(mask, merge, vs2, vs1, vl)` if merge is `vundefined()`
```

I suggest we could remove this too, and add one note at the end of this section: 
```
NOTE: compiler might change the tail/mask policy to agnostic policy if `merge` or `vd` operand is `vundefined()` value.
```

Reply:
done. thanks for you suggestion.